src/test/resources/readme_examples/li-service-a-readme.md

```diff
@@ -0,0 +1,104 @@
+# MicroProfile generated Application
+
+## Introduction
+
+MicroProfile Starter has generated this MicroProfile application for you.
+
+The generation of the executable jar file can be performed by issuing the following command
+
+    mvn clean package
+
+This will create an executable jar file **liberty.jar** within the _target_ maven folder. This can be started by executing the following command
+
+    java -jar target/liberty.jar
+
+
+### Liberty Dev Mode
+
+During development, you can use Liberty's development mode (dev mode) to code while observing and testing your changes on the fly.
+With the dev mode, you can code along and watch the change reflected in the running server right away; 
+unit and integration tests are run on pressing Enter in the command terminal; you can attach a debugger to the running server at any time to step through your code.
+
+    mvn liberty:dev
+
+
+
+To launch the test page, open your browser at the following URL
+
+    http://localhost:8181/index.html
```

Comment:
Should be 9080 instead of 8181

Reply:
Doesn't matter here - these are files for testing the parser....